data/plasma_protein_binding_rate_astrazeneca/meta.yaml

```diff
@@ -0,0 +1,52 @@
+---
+name: plasma_protein_binding_rate_astrazeneca
+description: |-
+    The human plasma protein binding rate (PPBR) is expressed as the percentage
+    of a drug bound to plasma proteins in the blood. This rate strongly affect a drug's
+    efficiency of delivery. The less bound a drug is, the more efficiently it can
+    traverse and diffuse to the site of actions.
+targets:
+    - id: rate_of_PPBR
+      description: percentage of a drug bound to plasma proteins in the blood
+      units: percentage
+      type: continuous
+      names:
+          - noun: human plasma protein binding rate (PPBR)
+          - noun: human plasma protein binding rate
+          - noun: PPBR
+          - noun: percentage of a drug bound to plasma proteins in the blood
+      uris:
+          - http://purl.jp/bio/4/id/201306028362680450
+          - http://www.bioassayontology.org/bao#BAO_0010135
+identifiers:
+    - id: SMILES
+      type: SMILES
+      description: SMILES
+    - id: Species
+      type: Other
+      names:
+          - species
+      description: species in which the measurement was carried out
+    - id: chembl_id
```

Comment:
@MicPie do we want to keep this for the prompts? will learning the IDs help the model?

Reply:
maybe not for the prompt but for merging it into the KG?

Reply:
yes, so shall we add an attribute that prevents sampling? Probably I'd anyhow merge on SMILES?

Reply:
cause currently we have no mechanism to prevent sampling of such identifiers

Reply:
yeah, very likely we will merge only in SMILES, and we could exclude cols with the id `chembl_id`.

Reply:
We could easily prevent sampling by not using the column in the template, or am I missing something?

Reply:
added flag for it